clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -125,150 +280,64 @@ export const CustomSelect = ({
   isSearchable,
   isClearable,
   size = "sm",
+  isMulti,
+  variant = "inline",
   ...props
-}: SelectProps & StringField) => {
```

Comment:
I'm probably going to introduce merge conflicts for you: https://github.com/ethyca/fides/pull/2568/files#diff-f59681eb1681d3b25307f6abef05b10c5926bafe6543643fcb07a95e5cd4e55fR115
Whether we even need that comes down to the overrides convo: https://ethyca.slack.com/archives/C043URV1KLL/p1676054836626409

Reply:
okay I think I solved the merge conflicts, but you may want to keep an eye on it...